### ISSUE TITLE
Issue restoring diffs with Restore-Dba-Backups

### ISSUE THREAD

**Issue by garethnewman** [sqlcollaborative/dbatools#3945]
# sql server 2016 dev edition - windows 10
# backup and restore with no recovery, later we want a diff
Backup-DbaDatabase -SqlInstance server1 -Database (Get-Content C:\temp\dbs.txt) -BackupDirectory "\\server2\Backup" -Type Full | Restore-DbaDatabase -SqlInstance server2 -NoRecovery
# make a diff backup and restore it
Backup-DbaDatabase -SqlInstance server1 -Database (Get-Content C:\temp\dbs.txt) -BackupDirectory "\\server2\Backup" -Type Diff | Restore-DbaDatabase -SqlInstance server2

# error thrown

The property 'Fullname' cannot be found on this object. Verify that the property exists and can be set.
At C:\Program Files\WindowsPowerShell\Modules\dbatools\0.9.395\allcommands.ps1:70899 char:17
+ ...             $full.Fullname = ($DatabaseHistory | Where-Object {$_.Typ ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : PropertyNotFound

# a backup is created and restored and then a diff is created so its the final restore that seems to have an issue

**Comment by Zhunya** [comment]
Please add `-Continue` switch!

**Comment by Stuart-Moore** [comment]
Use the -continue switch on the diff restore.

**Comment by garethnewman** [comment]
thanks guys, i'll try it.

**Comment by garethnewman** [comment]
I got a bunch of access denied messages but the diff restore worked. Thanks! :)

```WARNING: [10:58:19][New-DbaSqlDirectory] Failure | xp_create_subdir() returned error 5, 'Access is denied.'

**Comment by Zhunya** [comment]
That your server-side errors, please check if SQL Server service account 
has access to file system where your backup files stored

**Comment by garethnewman** [comment]
Yes I thought that but the restore worked so I figured it must have access?

**Comment by potatoqualitee** [comment]
I'm going to fix this issue with a better warning brb
